src/handlers.ts

```diff
@@ -8,6 +8,24 @@ import {
 
 export async function getRandomImage(req: Request, res: Response) {
   const { width, height } = req.query;
+  if (width && height) {
+    if (isNaN(Number(width)) || isNaN(Number(height))) {
+      return res
+        .status(400)
+        .json("The value of width and height should be a number");
+    }
+    if (Number(width) > 2000 || Number(height) > 2000) {
+      return res
+        .status(400)
+        .json("The value of width and height should be smaller than 2000");
+    }
+    if (Number(width) <= 0 || Number(height) <= 0) {
+      return res
+        .status(400)
+        .json("The value of width and height should be bigger than 0");
+    }
+  }
+
```

Comment:
Maybe you can extract this logic to another function like
```ts
function isNumberInRange(n: number, min: number, max: number)
```

I am not sure whether it's necessary. It is totally on you :)